en/docs/ipaas/micro-integrator/developing-integrations-with-integration-studio.md

```diff
@@ -0,0 +1,107 @@
+# Developing Integrations with Integration Studio
```

Comment:
In Choreo documentation, we follow a practice where we do not have titles in present continuous tense.

Shall we change this to "Develop Integrations with Integration Studio"? 

Let's change the file name too.